### PR TITLE
Creates icon package generator

### PR DESCRIPTION
**Pull request by shaneriley**
This creates a shell script and scaffolding files to automate the creation of new icon packages. With the script, you can copy your SVG file to the 'icons' directory, run the script with two or three arguments, and the required package files will be generated. As an example, you can run `bin/bootstrap_icon test.svg test 'test generation'` to generate the necessary files for a new icon package in 'packages/icon-test'. Details are in the readme, including instructions on how to perform a dry run.

### CODE REVIEW

File: scaffold/component.template.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  }
+
+  render() {
+    return icon;
+  }
+}
+
+customElements.define('ia-icon-${iconName}', IAIcon${className});
+
+export default IAIcon;`;
```
Comment (bfalling) [inline scaffold/component.template.js:31]: Too bad JS doesn't offer a built-in way to have indent-aware multiline strings.

Comment (shaneriley) [reply scaffold/component.template.js:31]: ??? That's what a template literal does. If you check out the file that is generated from this plain text, whitespace is retained.

Comment (bfalling) [reply scaffold/component.template.js:31]: No, I mean being able to have the delimiter(s) set an indent level that the multiline string processor will then consider a baseline, ignoring the initial whitespace, e.g. this example from Swift:
image

Comment (bfalling) [reply scaffold/component.template.js:31]: Then you wouldn't have to smash all your template code lines to the left, which makes it harder to parse, with respect to the surrounding code.

Comment (shaneriley) [reply scaffold/component.template.js:31]: Ah, a heredoc instead of a squiggly heredoc.

**Pull request merged**